files/organizations/deny_leaving_org.json.tpl

```diff
@@ -0,0 +1,16 @@
+{
+  "Version": "2012-10-17",
+  "Statement": {
+    "Sid": "DenyLeavingOrg",
+    "Effect": "Deny",
+    "Action": "organizations:LeaveOrganization",
+    "Resource": "*",
+    "Condition": {
```

Comment:
If you do not specify the `principal_exceptions`, will this still work? Have you ran a test plan? Because I think you will end up with an empty condition. Can you double check if that will be aproblem? Otherwise the if statement needs to include Condition I think

Reply:
I fixed the variable to have a default value and optional when not defined.